src/main/groovy/org/jetbrains/intellij/IntelliJPluginExtension.groovy

```diff
@@ -27,6 +29,7 @@ class IntelliJPluginExtension {
     private final Map<String, Object> systemProperties = new HashMap<>()
 
     String getType() {
+        if (version == null) return type ?: 'IC'
```

Comment:
You don't really use it when `localPath` is specified. I think the type should be `null` in this case.